internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -197,12 +200,14 @@ func verifyInputs(spec *loggingv1.ClusterLogForwarderSpec, status *loggingv1.Clu
 			badInput("inputspec cannot have a negative limit threshold")
 		case input.Receiver != nil && !extras[constants.VectorName]:
 			badInput("ReceiverSpecs are only supported for the vector log collector")
-		case input.Receiver != nil && input.Receiver.HTTP == nil:
-			badInput("ReceiverSpec must define an HTTP receiver")
+		case input.Receiver != nil && input.Receiver.HTTP == nil && input.Receiver.Syslog == nil:
```

Comment:
I don't think we want _both_ HTTP and Syslog defined for a receiver.

Reply:
Why not? They are intended for different things and serve different purposes.

Reply:
I'll defer to @alanconway for the final word, but I think it would complicate both upstream - k8s service - as well as downstream - vector xforms - plumbing if we try to support multiple protocols, or even multiple payload formats, over the same receiver.

Reply:
Ah true, for sure we do not want to have two protocols over the same receiver. For having two different receivers we should create two different inputs. Im going to fix this.